packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/test/Providers/Definitions/ModelReaderWriterContextDefinitionTests.cs

```diff
@@ -684,21 +688,25 @@ public void ValidateNestedFrameworkTypeHierarchy()
             var contextDefinition = new ModelReaderWriterContextDefinition();
             var attributes = contextDefinition.Attributes;
 
+            var expectedCount = 4;
             Assert.IsNotNull(attributes);
-            Assert.AreEqual(3, attributes.Count);
+            Assert.AreEqual(expectedCount, attributes.Count);
 
             // Should include both the collection model and the framework type it contains
             var buildableAttributes = attributes.Where(a => a.Type.IsFrameworkType && a.Type.FrameworkType == typeof(ModelReaderWriterBuildableAttribute)).ToList();
-            Assert.AreEqual(3, buildableAttributes.Count());
+            Assert.AreEqual(expectedCount, buildableAttributes.Count());
             Assert.AreEqual(
-                "typeof(global::Sample.Models.ComplexModel)",
```

Comment:
Is the attribute ordering being changed? We should make sure things are sorted the same.